### PR TITLE
Layout xml editing guidelines

### PR DESCRIPTION
**Pull request by DanJon91**
* Designtime attributes usage guidelines to make layout previews more informative
* Dimension resources usage guidelines to make layouts consistent and responsive

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/DanskeBank-MobEco/android-guidelines/pull/7" target="_blank">ReviewNinja

File: project_and_code_guidelines.md [hunk 1]
```diff
@@ -644,6 +644,106 @@ As a general rule you should try to group similar attributes together. A good wa
...
+#### 2.3.5.1 Use screen margin dimension resources
+
+Use sceen margin dimensions for container views that match screen size to better adapt to diferent screen configurations.
+
+| Dimension | Usage |
+| --- | ---|
+| `screen_margin_horizontal` | `Use to set horizontal margin to screen content` |
+| `screen_margin_vertical` | `Use to set vertical margin to screen content` |
+
+### 2.3.5 Use of dimension resources
```
Comment (Ainiuso) [inline project_and_code_guidelines.md]: !resolve the header text and number are already used.

File: project_and_code_guidelines.md [hunk 2]
```diff
@@ -644,6 +644,106 @@ As a general rule you should try to group similar attributes together. A good wa
...
+
+All components align to an 8dp square baseline grid. Iconography in toolbars align to a 4dp square baseline grid. Dimensions that can be reused must be stored in dimension resources, to keep projects visual design language consitent.
+
+#### 2.3.5.1 Use common dimension resources
+
+Use common dimension resources for view paddings and margins to make layouts consistent and better adapt to different screen configurations.
+
+| Margins | Paddings |
```
Comment (Ainiuso) [inline project_and_code_guidelines.md]: !fix something is wrong with table format

Comment (Ainiuso) [inline project_and_code_guidelines.md]: !fixed

File: project_and_code_guidelines.md (hunk 1 above)
Comment (Ainiuso) [inline project_and_code_guidelines.md]: !resolved

**Pull request merged**